test/fixtures/with-inline-plugins/.template-lintrc.js

```diff
@@ -1,9 +1,12 @@
-module.exports = {
+
+import inlineComponent from './rules/inline-component.js';
+
+export default {
   plugins: [
     {
       name: 'plugin1',
       rules: {
-        'inline-component': require('./rules/lint-inline-component')
```

Comment:
Any fixture directly importing a rule had to be converted to ESM since rules are now ESM.